tests/puppeteer/pages/FO/searchResults.js

```diff
@@ -0,0 +1,23 @@
+require('module-alias/register');
+const FOBasePage = require('@pages/FO/FObasePage');
+
+module.exports = class SearchResults extends FOBasePage {
```

Comment:
```suggestion
module.exports = class SearchResultsPage extends FOBasePage {
```

Reply:
I think that in all the pages, the class name does not contain Page.

Reply:
Ok :+1: